src/box/lua/config/applier/box_cfg.lua

```diff
@@ -158,20 +158,21 @@ local function missing_names_is_empty(missing_names, replicaset_name)
            table.equals(missing_names._peers, {})
 end
 
-local function names_on_rw_transit()
+local function names_try_set()
```

Comment:
> NO_CHANGELOG=bugfix

The code with the bug was released as 3.0.0-beta1, so it worth to write a changelog entry.

Reply:
Done